CHANGELOG.md

```diff
@@ -4,6 +4,24 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/) and this project adheres to [Semantic Versioning](http://semver.org/).
 
+### 0.18.0
+
+**Released**: 2021.09.13
+
+**Commit Delta**: [Change from 0.17.1 release](https://github.com/plus3it/tardigrade-ci/compare/0.17.0..0.18.0)
+
+**Summary**:
+
+*   The new environment variable "PROVIDER_ALIAS" can be used to specify the 
+    name of a Terraform provider alias.  Used as a value for the new "--alias" 
+    command line option, it instructs the automated integration test tool
+    to create the aliased provider information in preparation for a test run.
+
+*   When running Terraform tests, the Firehose service will now be provided
+    by moto and not LocalStack.
+
+*   No changes to tool versions.
```

Comment:
There are actually a number of changes to tool versions, since those are merged to master, even though those changes are not part of _this_ branch. You can check by comparing your branch against the last tag, instead of master...

* `https://github.com/plus3it/tardigrade-ci/compare/0.17.1...kbalk:add-provider-alias#files_bucket`

Reply:
Then there is a problem with the way we're listing tool updates in the change log if tool changes aren't always mentioned unless there are non-tool changes.  I'll add the tool updates.